### PR TITLE
Fixes and cleanup for logging.

### PR DESCRIPTION
**Pull request by uomo91**
## About The Pull Request

Author of https://github.com/Skyrat-SS13/Skyrat-tg/pull/1704 and https://github.com/Skyrat-SS13/Skyrat-tg/pull/1713 here.

This PR makes subtle and subtler emotes display properly in individual logging (so they don't look like normal emotes or like regular say logs), and fixes a preexisting runtime with LOG_SUBTLER for game.log.
This also by necessity adds a config flag in config.txt for logging subtler emotes in game.txt or not. I've set it to false for now, not sure how you guys want to handle that. If this PR is merged as-is, this will allow admins to see subtler emotes in ongoing rounds via player's individual logging panels, but subtler emotes will _not_ be permanently stored in game.txt. Uncomment the config option to change this.

I don't know if this meets your modularization guidelines or not, if there are any problems I'll try to restructure as best possible.
Just trying to see this code through to every downstream that took it.

## Why It's Good For The Game

Trying to make the logging code as consistent an experience for admins as I can.

## Changelog
:cl:
tweak: Subtle and subtler emotes are clearly labeled in individual logging.
fix: Fixed a runtime for subtler emotes.
admin: Adds a config option for logging subtler emotes in game.txt or not.
/:cl:

### CODE REVIEW

File: code/modules/mob/mob_helpers.dm [hunk 1]
```diff
@@ -492,6 +492,8 @@
 			colored_message = "(ASAY) [colored_message]"
 		if(LOG_EMOTE)
 			colored_message = "(EMOTE) [colored_message]"
+		if(LOG_SUBTLER)
+			colored_message = "(EMOTE) (SUBTLER) [colored_message]"
```
Comment (Azarak) [inline code/modules/mob/mob_helpers.dm:497]:
```suggestion
		//SKYRAT EDIT ADDITION BEGIN
		if(LOG_SUBTLER)
			colored_message = "(EMOTE) (SUBTLER) [colored_message]"
		//SKYRAT EDIT ADDITION END
```

File: config/config.txt [hunk 2]
```diff
@@ -103,6 +103,9 @@ LOG_WHISPER
 ## log emotes
 LOG_EMOTE
 
+## log subtler emotes
+#LOG_SUBTLER
+
```
Comment (Azarak) [inline config/config.txt]: Could you move those to inside "config/skyrat/skyrat_config.txt" ?

File: code/game/atoms.dm [hunk 3]
```diff
@@ -1341,6 +1341,8 @@
 			log_whisper(log_text)
 		if(LOG_EMOTE)
 			log_emote(log_text)
+		if(LOG_SUBTLER)
+			log_subtler(log_text)
```
Comment (Azarak) [inline code/game/atoms.dm:1346]:
```suggestion
		//SKYRAT EDIT ADDITION BEGIN
		if(LOG_SUBTLER)
			log_subtler(log_text)
		//SKYRAT EDIT ADDITION END
```

File: code/__HELPERS/_logging.dm [hunk 4]
```diff
@@ -134,6 +134,10 @@
 	if (CONFIG_GET(flag/log_emote))
 		WRITE_LOG(GLOB.world_game_log, "EMOTE: [text]")
 
+/proc/log_subtler(text)
+	if (CONFIG_GET(flag/log_subtler))
+		WRITE_LOG(GLOB.world_game_log, "SUBTLER EMOTE: [text]")
+
```
Comment (Azarak) [inline code/__HELPERS/_logging.dm]: Could you move this to somewhere inside "code\__HELPERS\~skyrat_helpers"? Be it in unsorted or you could make a "logging.dm"

File: code/controllers/configuration/entries/general.dm [hunk 5]
```diff
@@ -77,6 +77,8 @@
 
 /datum/config_entry/flag/log_emote	// log emotes
 
+/datum/config_entry/flag/log_subtler // log subtler emotes
+
```
Comment (Azarak) [inline code/controllers/configuration/entries/general.dm]:
```suggestion
/datum/config_entry/flag/log_subtler // log subtler emotes //SKYRAT EDIT ADDITION

```

File: config/config.txt (hunk 2 above)
Comment (uomo91) [reply config/config.txt]: Done.

File: code/__HELPERS/_logging.dm (hunk 4 above)
Comment (uomo91) [reply code/__HELPERS/_logging.dm]: Also done.

**Review by Azarak — APPROVED** [review]

**Comment by uomo91** [comment]
One remaining inconsistency is that subtle (not subtler) emotes for some reason show up in say tab, as well as in the emote tab and the show all tab. I'm not sure how to fix this, LOG_SUBTLE might need to be created.

**Pull request merged**